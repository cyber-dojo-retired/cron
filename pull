#!/bin/sh
set -e

# This file is run as a cron job and cannot have an extension.
# Viz it must be called pull and not, say, pull.sh

wget --quiet \
  -O- \
  --post-data='{}' \
  --header='Content-Type:application/json' \
  http://puller:5017/pull \
  | tee /tmp/pull.log
