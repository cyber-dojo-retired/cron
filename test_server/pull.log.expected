{"pull":["cyberdojofoundation/csharp_nunit","cyberdojofoundation/gcc_assert","cyberdojofoundation/gpp_assert","cyberdojofoundation/java_junit","cyberdojofoundation/python_unittest","cyberdojofoundation/ruby_mini_test","cyberdojofoundation/ruby_test_unit"]}